### ISSUE TITLE
0-bootstrap group names don't align with documentation

### ISSUE THREAD

**Issue by arueth** [terraform-google-modules/terraform-example-foundation#1092]
### TL;DR

- In [0-bootstrap/terraform.examples.tfvars](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/terraform.example.tfvars#L40), group names don't align with documentation on the [Cloud Architecture Center](https://cloud.google.com/architecture/security-foundations/authentication-authorization#groups_for_access_control)
- There is an audit_data_users group in the Terraform that doesn't exist in the documentation. 
- There is a gcp-kms-admin group in the documentation that doesn't exists in the Terraform.

### Expected behavior

Terraform and documentation should match.

### Observed behavior

Discrepancies between Terraform and documentation

### Terraform Configuration

```hcl
#  Optional - for enabling the automatic groups creation, uncoment the groups
#  variable and update the values with the desired group names
# groups = {
#   create_groups = true,
#   billing_project = "billing-project",
#   required_groups = {
#     group_org_admins           = "group_org_admins_local_test@example.com"
#     group_billing_admins       = "group_billing_admins_local_test@example.com"
#     billing_data_users         = "billing_data_users_local_test@example.com"
#     audit_data_users           = "audit_data_users_local_test@example.com"
#     monitoring_workspace_users = "monitoring_workspace_users_local_test@example.com"
#   },
#   optional_groups = {
#     gcp_platform_viewer      = "gcp_platform_viewer_local_test@example.com"
#     gcp_security_reviewer    = "gcp_security_reviewer_local_test@example.com"
#     gcp_network_viewer       = "gcp_network_viewer_local_test@example.com"
#     gcp_scc_admin            = "gcp_scc_admin_local_test@example.com"
#     gcp_global_secrets_admin = "gcp_global_secrets_admin_local_test@example.com"
#     gcp_audit_viewer         = "gcp_audit_viewer_local_test@example.com"
#   }
# }
```

### Terraform Version

```sh
Terraform v1.7.1
on linux_amd64
```

**Comment by fmichaelobrien** [comment]
reference similar in 1-environments
https://github.com/terraform-google-modules/terraform-example-foundation/pull/1184/files#diff-6ddd19538b7d3c7bf4be3478412858533b6ef360b21313b232662ac7be688b51L24
